Classes/Officer.py

```diff
@@ -434,6 +434,13 @@ async def log_message_activity(self, msg, send_time=None):
                 (msg.id, msg.channel.id, self.id, string_send_time),
             )
 
+    async def renew_time(self):
+        """Inserts a 1 second on duty session into the Officer's logbook to trick the inactivity hook"""
+        start_time = datetime.timestamp(datetime.utcnow())
+        end_time = start_time + 1
+
+        await self.log_time(start_time, end_time)
```

Comment:
Adding a second to their on duty time is a very hacky way to increase their time, we could have a separate table for storing the last time renewal.

Also, adding a separate bot command to call this on an officer, like =renew_time would be very handy.

Reply:
I have rewritten things to instead use a separate column for the renewed_time in Officers. This will require the database to be updated as shown in database.sql.